packages/boost-histogram/test_boost_histogram.py

```diff
@@ -0,0 +1,41 @@
+import pytest
```

Comment:
Does this have to be imported inside the function, then?

Reply:
Yeah. We usually use `unittest.assertThrows` since we don't want to install `pytest`.

Reply:
Or more accurately, `unittest.TestCase().assertRaises`.